components/Skeleton/Skeleton.jsx

```diff
@@ -1,85 +1,45 @@
 import PropTypes from 'prop-types';
-// eslint-disable-next-line import/no-cycle
-import SkeletonTypes from './SkeletonTypes';
-import { spacing, baseFontSize as defaultBaseFontSize } from '../shared/theme';
-import {
-  SkeletonRect,
-  SkeletonCircle,
-  SkeletonText,
-  SkeletonButton,
-  SkeletonTag,
-} from './sub-components';
+import * as Styles from './Skeleton.styles';
 
-const Skeleton = props => {
-  const { type, size, theme, height, width } = props;
+const Skeleton = ({ type, ...props }) => {
+  switch (type) {
+    case 'circle':
+      return <Styles.Circle {...props} />;
 
-  const rectProps = {
-    height,
-    width,
-  };
-  const circleProps = {
-    height,
-    width,
-  };
+    case 'text':
+      return <Styles.Text {...props} />;
 
-  const textProps = {
-    height,
-    width,
-  };
+    case 'button':
+      return <Styles.Button {...props} />;
 
-  const buttonProps = {
-    size,
-    theme,
-    width,
-  };
+    case 'tag':
+      return <Styles.Tag {...props} />;
 
-  const tagProps = {
-    size,
-    theme,
-    width,
-  };
-
-  return (
-    <>
-      {type === 'rect' && <SkeletonRect {...rectProps} />}
-      {type === 'circle' && <SkeletonCircle {...circleProps} />}
-      {type === 'text' && <SkeletonText {...textProps} />}
-      {type === 'button' && <SkeletonButton {...buttonProps} />}
-      {type === 'tag' && <SkeletonTag {...tagProps} />}
-    </>
-  );
+    default:
+      return <Styles.Rect {...props} />;
+  }
 };
 
 Skeleton.defaultProps = {
   type: 'rect',
+  as: 'div',
   width: '100%',
   height: '14px',
   size: 'medium',
-  theme: {
-    baseFontSize: defaultBaseFontSize,
-    spacing,
-  },
 };
 
 Skeleton.propTypes = {
-  /** Sets the component behavior */
```

Comment:
These comments about the properties relate to their description when accessing the documentation (yarn storybook). Deleting them implies in making it difficult to understand the behavior of the components for those who are interested in using it.

Reply:
Oh I was not aware of that. I will restore the comments. I apologize for the oversight.